content/en/data_security/guide/tls_ciphers_deprecation.md

```diff
@@ -4,7 +4,9 @@
 
 As part of its ongoing commitment to the security and protection of its customer's data, Datadog is rolling out a more modern cryptographic engine across its systems which imposes some changes to the configurations it can accept.
 
-Beginning **April 1st, 2024**, Datadog is disabling support for the following cipher suites across its public-facing applications. If you use unsupported clients to connect to Datadog after the older protocols are disabled, you will receive connection error messages.
+Beginning **April 1st, 2024**, Datadog disabled support for the following cipher suites across its public-facing applications. If you use unsupported clients to connect to Datadog after the older protocols are disabled, you will receive connection error messages.
+
+Starting **July 1st, 2024**, Datadog will force HTTP to HTTPS redirection for all intake endpoints, using an HTTP 308 response code. This change is necessary as some intake endpoints still accept plain HTTP payloads to maintain compatibility with older API clients. Ensure that your application clients can follow these redirection codes, or update their configuration to utilize HTTPS. Otherwise, you might receive connection error messages.
```

Comment:
some very opinion-oriented feedback:
- I wonder about the usage of "beginning" vs. "starting"—I do think lines 7 and 9 should have some kind of differentiation, as 7 discusses something that has already happened vs. 9 discussing something that will happen in the future, but I don't know that beginning vs. starting is the best way to do this.
- I don't love that line 7 mentions "the following cipher suites," but then what immediately follows is an unrelated sentence. Perhaps refer instead to "the cipher suites in the following section"?
- Alternatively, maybe we could put line 9 in an alert-warning box and slap it at the top of this page?

approving since I don't really have any hard lines here; the choice is yours.

Reply:
the second paragraph should have been removed.